.github/workflows/generate-summary.yaml

```diff
@@ -429,7 +429,7 @@ jobs:
           printf 'A list of all unresolved "internal compiler error", "Segmentation fault", "test for excess errors" failures present at this hash\n\n' >> trimmed_comment.md
```

Comment:
Could you also change this for the `Trim issue length' step

Reply:
Not entirely sure what it means. Do you want the statement to be changed into something like 
"A list of trimmed unresolved "internal compiler error", "Segmentation fault", "test for excess errors" failures present at this hash\n\n" if it's trimmed instead of the original statement?

Reply:
I mean for the actual issue creation, not the comment creation. So this https://github.com/patrick-rivos/gcc-postcommit-ci/blob/main/.github/workflows/generate-summary.yaml#L444-L449 and https://github.com/patrick-rivos/gcc-postcommit-ci/blob/main/.github/workflows/generate-summary.yaml#L740-L745

Reply:
oh I see the command change from wc -l to stat -c